### PR TITLE
[Merged by Bors] - refactor(Probability/Kernel/CondCdf): mv tendsto_of_antitone

### PR DESCRIPTION
**Pull request by mo271**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Topology/Algebra/Order/MonotoneConvergence.lean [hunk 1]
```diff
@@ -310,6 +310,12 @@ theorem iInf_eq_of_tendsto {α} [TopologicalSpace α] [CompleteLinearOrder α] [
   tendsto_nhds_unique (tendsto_atTop_iInf hf)
 #align infi_eq_of_tendsto iInf_eq_of_tendsto
 
+theorem tendsto_of_antitone {ι α : Type*} [Preorder ι] [TopologicalSpace α]
```
Comment (RemyDegenne) [inline Mathlib/Topology/Algebra/Order/MonotoneConvergence.lean:313]: Can you put that next to `tendsto_of_monotone`? Or is there a good reason for putting it here instead?

Comment (mo271) [reply Mathlib/Topology/Algebra/Order/MonotoneConvergence.lean]: That makes, sense, done!

**Comment by RemyDegenne** [comment]
Thanks!
bors r+

**Comment by mo271** [comment]
fixed the merge conflict

**Pull request closed without merging**